.github/workflows/check.yml

```diff
@@ -7,7 +7,7 @@ jobs:
       matrix:
         script:
           - ./code-analysis/scripts/check.sh
-          - ./documentation/scripts/build.sh
+          - ./code-analysis/crates/solidity/outputs/documentation/scripts/build.sh
```

Comment:
I agree with moving the generated grammar/spec into an `outputs/*` folder. I was originally planning on doing this in #76, where:

1. Solidity-specific generated markdown can live in `outputs/documentation`.
2. The containing `mkdocs` build and other content will remain in `root/documentation` as-is.
3. And the second can reference the first (as a sub-directory) from its new location during the build.

If I may suggest, can we delay that move, until we get to #76? This requires some changes to the tree structure we generate, that will be done for versioning anyways. I plan to start on this after testing is complete.

The changes in this PR makes the whole `mkdocs` project exclusive to Solidity output. It is intended to host not only generated Solidity spec, but also our project internals docs, and different notes/marketing unrelated to the language codegen itself.

If there is anything that require us to do that move now/sooner than later, I'm happy to reprioritize to unblock you.